thirdparty/patches/grpc-python.patch

```diff
@@ -1,23 +1,63 @@
 diff --git third_party/py/python_configure.bzl third_party/py/python_configure.bzl
 --- third_party/py/python_configure.bzl
 +++ third_party/py/python_configure.bzl
-@@ -163,1 +163,1 @@
+@@ -177,7 +177,7 @@ def _get_bash_bin(repository_ctx):
```

Comment:
Can you tell me why we need to change this file?

Reply:
We updated our gRPC dependency, so these patches were no longer applying to the correct lines, i.e. the correct code.